src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -245,6 +245,27 @@ export class PatientViewPageStore
 
     }, []);
 
+    readonly MDAndersonHeatMap = remoteData({

```

Comment:
This is sort of nitpicky but, it seems like you're not actually using this fetched data, you just want to know it exists. So to keep things simple and direct, I'd prefer to see this be called `MDAndersonHeatMapAvailable` and have it return the boolean `fileContent.length > 0`, default `false`